cSpell.json

```diff
@@ -0,0 +1,1376 @@
+        "Unprovision",
+        "unprovisioned",
+        "Unprovisioning",
+        "Unprovisions",
+        "Unregistering",
+        "Unregisters",
+        "unsubstituted",
+        "untagged",
+        "unvalidated",
+        "unwrapkey",
+        "updatecertificate",
+        "updatehostname",
+        "updateproperties",
+        "upgradeos",
+        "uploadbatchservicelogs",
+        "uploadcertificate",
+        "upns",
+        "upsell",
+        "Upsert",
+        "upserting",
+        "usagescenarios",
+        "usererror",
+        "userflows",
+        "userpreference",
+        "usersetting",
+        "USQL",
+        "vaultconfig",
+        "vaultstorageconfig",
+        "vaultusages",
+        "Vcore",
+        "Vcores",
+        "Vertica",
+        "vhds",
+        "videosearch",
+        "virtualip",
+        "virtualmachine",
+        "virtualmachines",
+        "virtualnetworkgateways",
+        "virtualnetworkrules",
+        "virtualnetworks",
+        "viruela",
+        "visualsearch",
+        "visualstudio",
+        "vlan",
+        "vlanid",
+        "VM's",
+        "vmextension",
+        "VMILR",
+        "vmimage",
+        "vmname",
+        "vmsizes",
+        "VMSS",
+        "VMSSVM",
+        "vmware",
+        "vnet",
+        "vnetid",
+        "Vnets",
+        "Vpnclient",
+        "vpnconfiguration",
+        "vpndeviceconfigurationscript",
+        "vray",
+        "VSTS",
+        "VXLAN",
+        "waagent",
+        "WADL",
+        "waitforjobcompletion",
+        "waitingforstarttask",
+        "WANAs",
+        "WANs",
+        "WASB",
+        "WDATP",
+        "webapplicationfirewall",
+        "webapps",
+        "Webchat",
+        "webhdfs",
+        "webhook",
+        "webhooks",
+        "webhostingplans",
+        "webjob",
+        "webjobs",
+        "webproxy",
+        "websearch",
+        "webservices",
+        "Webspace",
+        "webtest",
+        "webtests",
+        "westcentralus",
+        "westeurope",
+        "westindia",
+        "Westlake",
+        "westus",
+        "whitelisted",
+        "Whitelisting",
+        "Whois",
+        "Widevine",
+        "windowsazuremediaservices",
+        "wlmengg",
+        "Workernode",
+        "workloadmonitor",
+        "workspace's",
+        "workspaces",
+        "wrapkey",
+        "WSFC",
+        "WTGCTBG",
+        "XBing",
+        "Xero",
+        "XSMB",
+        "YYMMDD",
+        "Zabbix",
+        "Zilla",
+        "ziplist",
+        "Zoho",
+        "zset"
+    ],
+    "flagWords": [],
+    "ignorePaths": [
+        "node_modules/**",
+        "**/examples/**",
+        "**/cSpell.json"
+    ],
+    "ignoreRegExpList": [
+        "email",
+        "Base64"
+    ],
+    "minWordLength": 4,
+    "dictionaries": [
+        "companies",
+        "softwareTerms",
+        "html",
+        "typescript",
+        "python",
+        "node",
+        "go",
+        "java",
+        "csharp"
+    ],
+    "overrides": [
```

Comment:
@jianghaolu these would be examples of suppression. I tried the nested config files, but these seemed to work better.

Reply:
So I assume there isn't a way to do it in individual spec folders. Let me bring this to the team.

Reply:
It takes a glob/regex, so you could relace it with `somefolder/**` to apply the overrides to a folder

Reply:
These examples I did at the file level with the assumption that these typos are breaking changes to fix in the current version, but you'd want it flagged in the next version so it would be fixed there

Reply:
Oh I was actually asking if we can move this section to a separate `cSpell.partial.json` in `somefolder/`. As this list grows it may be easier to organize that way.

Reply:
Ah, I believe that you can just have nested cSpell.json files, but I can't remember if I actually ran into an issue with that or that I thought it might be more confusion having other non-swagger json files mixed in